packages/circuit-ui/components/Grid/Row/__snapshots__/Row.spec.tsx.snap

```diff
@@ -3,6 +3,8 @@
 exports[`Row should render with default styles 1`] = `
 .circuit-0 {
   position: relative;
+  margin-left: calc(-16px / 2);
+  margin-right: calc(-16px / 2);
```

Comment:
This addition to the default styles is caused by the consistent use of `composeBreakpoints` in all grid components. It has no effect since it's overridden by the breakpoint styles below which have higher specificity.